alias.c

```diff
@@ -681,17 +648,23 @@ bool mutt_addr_is_user(struct Address *addr)
   return false;
 }
 
-void mutt_alias_free(struct Alias **p)
+void mutt_alias_free(struct Alias *p)
```

Comment:
Please can you change this back to `void mutt_alias_free(struct Alias **p)`
(to keep it the same as all our other 'free' functions -- free and NULL)

Reply:
@flatcap done.